blueocean-dashboard/src/main/less/run-pipeline.less

```diff
@@ -0,0 +1,35 @@
+.jdl-table {
+  .run-pipeline {
+    z-index: 2000;
+    font-size: 10px;
+    padding-left: 5px;
+    padding-top: 2px;
+    border-radius: 50%;
+    width: 20px;
+    height: 20px;
+    background-image: url('./icons/run.svg');
+    background-size: cover;
+  }
+
+  .run-pipeline:hover {
+    background-image: url('./icons/run-darkened.svg'); // TODO: nicer if it could be done via a data-uri transform
+  }
+
+  .run-pipeline-toast {
+    position: fixed;
+    left: 15px;
+    bottom: 15px;
+
+    // Need to set higher specificity rules for some of the
+    // Toast styles because they are being overridden by other
+    // styles in JDL (table styles).
+    .toast {
+      a.action {
+        color: @blueocean-blue;
+      }
+      .text {
+        white-space: normal;
+      }
```

Comment:
Could you explain the changes in the styles here? I think we want to use the same blue color for all links in the Toast, so if the design spec changed, I think we should update the JDL. Similar as far as far the text, there were some specific rules in the JDL about max width of text.